va/va.go

```diff
@@ -133,6 +135,15 @@ func initMetrics(stats metrics.Scope) *vaMetrics {
 			Help: "Number of HTTP-01 redirects followed",
 		})
 	stats.MustRegister(http01Redirects)
+	caaCounter := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "caa_sets_processed",
+		Help: "A counter of CAA sets processed labelled by result",
+	}, []string{"result"})
+	stats.MustRegister(caaCounter)
+	ipv4FallbackCounter := prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "tls_alpn_ipv4_fallback",
+		Help: "A counter of IPv4 fallbacks during TLS ALPN validation",
+	})
```

Comment:
Is this missing a `stats.MustRegister`?

Reply:
Also makes me think there's unit test coverage for this stat missing. Maybe a good follow-up issue.